campos_potenciais2.py

```
@@ -0,0 +1,278 @@
+DIST_GOAL_STOP = 0.3 
+
+
+def angle_diff(a, b):
+    """Retorna diferença angular (a - b) normalizada para [-pi, pi]."""
+    d = a - b
+    return np.arctan2(np.sin(d), np.cos(d))
+
+def clip(x, a, b):
+    return max(a, min(b, x))
+
+def transform_point_matrix(sim, mat4, point3):
```

Ruff (0.13.3)

Unused function argument: `sim`

(ARG001)